__tests__/endpoint-requests.test.js

```diff
@@ -169,3 +169,66 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("status: 200, should post an object to comments with the specified properties", () => {
```

Comment:
just check the description - you're expecting a 201 but you've written 200 here ( 201 is the correct status )